Week1/homework/js-exercises/dogGallery.js

```diff
@@ -0,0 +1,59 @@
+const XMLBtn = document.getElementById('btn1');
+const AxiosBtn = document.getElementById('btn2');
+
+function xhrMethod() {
+    const xhr = new XMLHttpRequest();
+    const li = document.createElement('li');
+    const image = document.createElement('img');
+    const ul = document.getElementById('list');
+    const url = 'https://dog.ceo/api/breeds/image/random'
+    xhr.open('GET', url);
+    xhr.send();
+    xhr.onreadystatechange = processRequest;
+
+    ul.appendChild(li.appendChild(image))
```

Comment:
Does this work? I guess it should be 
```
li.appendChild(image)
ul.appendChild(li)
```
as two separate lines.

Reply:
Yeah, it's working. Also it's possible to write it in two lines